README.md

```diff
@@ -17,94 +25,136 @@
 
 ### Chapter 2: Getting Started with Dash
 
- - Structure of a Dash app: data, layout, callback
- - Installation
+ - Structure of a Dash app
+ - Installation and setup
+ - Create your first Dash app
+ - Live updates and debugging
```

Comment:
these two topics seem a bit advanced to be talked about so early in the curriculum. I'm speaking solely from my experience: live updates and debugging were not easy for me to grasp when I learned it several months after starting with Dash. I'm worried our readers would have a harder experience than mine if they learned these topics so early. I was thinking maybe include this in part III. @firasm were you envisioning just an intro to these topics here? Any particular reason you wanted it in section 2?

Reply:
I should probably clarify - at this point, what we should teach students the students are things like:

- how to stop your app (and why you should) (Ctrl+C)
- how to update your app (Ctrl+C and then re-run)
- how to turn on "live updating" (with debug=True) and what this means (i.e. what happens if you have a syntax error and your app crashes)

Those are all I want to teach them at this point, and further exploration can come later (if at all).

Reply:
Thanks for clarifying. That makes sense to me.

Reply:
updated readme file with this.